src/platyPS/platyPS.psm1

```diff
@@ -1512,7 +1512,20 @@ function GetMarkdownFilesFromPath
         }
     }
 
-    return $MarkdownFiles
+    $MarkdownFilesFiltered = @()
+
+    If ($IncludeModulePage) {
+        $MarkdownFilesFiltered = $MarkdownFiles
+    } Else {
+        ForEach ($File in $MarkdownFiles) {
+            $Matches = [regex]::Match((Get-Content $File.FullName -Raw), "# $($File.BaseName) Module").Captures.Groups
```

Comment:
I don't understand that line, can you add a comment that explains the intention here?